From ba0f6058d1f25b2b60fc31ab2656bf12a71ffdab Mon Sep 17 00:00:00 2001
From: Lu Chong <Chong.Lu@windriver.com>
Date: Tue, 5 Nov 2013 17:32:56 +0800
Subject: [PATCH] ppp: Fix compilation errors in Makefile

Make can't exit while compilation error occurs in subdir for plugins building.

Upstream-Status: Pending

Signed-off-by: Lu Chong <Chong.Lu@windriver.com>
---
 pppd/plugins/Makefile.linux          |    1 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pppd/plugins/Makefile.linux b/pppd/plugins/Makefile.linux
index 0a7ec7b..2a2c15a 100644
--- a/pppd/plugins/Makefile.linux
+++ b/pppd/plugins/Makefile.linux
@@ -20,7 +20,7 @@ include .depend
 endif
 
 all:	$(PLUGINS)
-	for d in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$d all; done
+	for d in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$d all || exit 1; done
 
 %.so: %.c
 	$(CC) -o $@ $(LDFLAGS) $(CFLAGS) $^
-- 
1.7.9.5

